test/000_fullchain.test.ts

```diff
@@ -297,10 +298,125 @@ describe('Integration tests', function () {
                     frozenChanges: expectedFrozenChanges,
                 });
                 for (let i = 0; i < workerNumber; i++) {
-                    if (workerNumber == 1) {
-                        expect(await iexecPoco.viewScore(workers[i].address)).to.be.equal(0);
-                    }
+                    expect(await iexecPoco.viewScore(workers[i].address)).to.be.equal(
+                        workerNumber == 1 ? 0 : 1,
+                    );
+                }
+            });
+        }
+        for (let workerNumber = 4; workerNumber < 6; workerNumber++) {
+            // Worker1 will contribute badly
+            it(`[7.${workerNumber - 3}] No sponsorship, no beneficiary, no callback, no BoT, up to ${workerNumber} workers with 1 bad actor`, async function () {
+                const volume = 1;
+                const workersAvailable = [worker1, worker2, worker3, worker4, worker5];
+                const { resultDigest: badResultDigest } = buildUtf8ResultAndDigest('bad-result');
+                const losingWorker = worker1;
+                const winningWorkers = workersAvailable.slice(1, workerNumber);
+                let workers = [{ signer: worker1, resultDigest: badResultDigest }];
+                for (const worker of winningWorkers) {
+                    workers.push({ signer: worker, resultDigest: resultDigest });
+                }
```

Comment:
It seems to me more like `contributions` not `workers`
```suggestion
                for (const worker of winningWorkers) {
                    contributions.push({ signer: worker, resultDigest: resultDigest });
                }
```

Reply:
Done [here](https://github.com/iExecBlockchainComputing/PoCo/pull/160/commits/715c453c77ebbf3915c88e780564b53129cf4d78)